pvacseq/lib/main.py

```diff
@@ -237,7 +237,8 @@ def main(args_input = sys.argv[1:]):
         sys.exit("The fasta file is empty. Please check that the input VCF contains missense, inframe indel, or frameshift mutations.")
 
     tmp_dir = os.path.join(args.output_dir, 'tmp')
-    os.makedirs(tmp_dir)
+    if not os.path.exists(tmp_dir):
+        os.makedirs(tmp_dir)
```

Comment:
There's an [_exist\_ok_](https://docs.python.org/3/library/os.html#os.makedirs) parameter for `os.makedirs()` that'll let you do this in one line (not really an issue though).